armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -529,6 +529,20 @@ def test_specifyTargetComponentRuntimeErrorSecond(self):
             the_exception = cm.exception
             self.assertEqual(the_exception.error_code, 3)
 
+    def test_specifyTargetComponentBlockWithMultipleFlags(self):
+        # build block for testing
```

Comment:
you could update this comment to say why this block is special ( block has both fuel and poison flags, and components)